lib/views/screens/home/transactions_screen.dart

```diff
@@ -180,12 +188,13 @@ class _TransactionsSheetState extends State<TransactionsSheet> {
                         padding: const EdgeInsets.only(
                             bottom: kFloatingActionButtonMargin + 100),
                         elements: transactionsList,
-                        groupBy: (element) => element.toDateOnly(),
+                        groupBy: (element) => element.toUSDate(),
                         groupComparator: (value1, value2) =>
-                            value1.compareTo(value2),
+                            value2.compareTo(value1),
                         itemComparator: (item1, item2) =>
-                            item1.toTime().compareTo(item2.toTime()),
+                            item1.timestamp.compareTo(item2.timestamp),
                         order: GroupedListOrder.DESC,
```

Comment:
This was in ascending order and didn't make sense to me. Swapped it back to descending for now.